rustls/src/common_state.rs

```diff
@@ -345,31 +345,17 @@ impl CommonState {
     }
 
     /// Like send_msg_encrypt, but operate on an appdata directly.
-    fn send_appdata_encrypt(&mut self, payload: OutboundChunks<'_>, limit: Limit) -> usize {
```

Comment:
I like the direction here, but it would be nice to make these changes in smaller commits so that it's more obvious when reviewing that each step is correct.

Reply:
Yes, please split it more. Suggest doing one commit per function where you are removing the `limit` argument.

(Also please keep the first line of commit messages below 80 or so chars.)